packages/content-fetch/src/request_handler.ts

```diff
@@ -147,11 +148,29 @@ export const contentFetchRequestHandler: RequestHandler = async (req, res) => {
       logRecord.error = 'unknown error'
     }
 
+    // capture error event
+    users.forEach((user) => {
+      analytics.capture({
+        distinctId: user.id,
+        event: 'content-fetch-failure',
```

Comment:
Not sure why we picked it but we seem to use underscores so this would be `content_fetch_failure`